lib/ex_doc/formatter/html/templates/css/style.css

```diff
@@ -404,9 +410,9 @@ li.r2 {
 }
 
 div.breadcrumbs {
-  padding: 3px 6px;
+  padding: 3px 15px;
   background-color: #EEE;
-  margin: 0 -15px 10px;
+  margin: 0 -28px 10px;
```

Comment:
yes, but in https://github.com/eksperimental/ex_doc/blob/fix2/lib/ex_doc/formatter/html/templates/css/style.css#L73-L75 I have increased the margin.
What broweser gave you that problem? Did you click on the breadcumb links for that to happen? if so, it's a bug that is unrelated to this change and it's happening even before these changes: https://github.com/elixir-lang/ex_doc/issues/254